src/rpc/README.txt

```diff
@@ -0,0 +1,60 @@
+--------------------------------------------------------------------------------
+Python -> OCaml RPC
+--------------------------------------------------------------------------------
+
+The mechanism implemented here and in `ocaml.py` in pysemgrep allows us to
+incrementally migrate functionality from Python to OCaml in support of our
+osemgrep effort. Instead of being limited to the one main call out to
+semgrep-core, we can now make additional calls from arbitrary points in the
+Python code.
+
+To add a function:
+
+- Update the `function_call` and `function_return` types in
```

Comment:
Don't we have a ATD spec just for semgrep-core ? I thought this ATD was mainly for the output of Semgrep, and this is internal stuff?

Reply:
I believe that this file is used for both. We have a lot of internal-only stuff in that file already that has to do with the interface between pysemgrep and semgrep-core.

Reply:
I initially made a separate file for this RPC but then realized that it would be useful to have access to the types defined in the main file.

Reply:
yes right now atd does not have a module system (one of the TODO of martin), so the spec of the CLI output and core output is in the same file, because they share types.